spinoso-string/src/enc/utf8/impls.rs

```diff
@@ -0,0 +1,181 @@
+use alloc::borrow::Cow;
+use alloc::string::String;
+use alloc::vec::Vec;
+use core::borrow::{Borrow, BorrowMut};
+#[cfg(feature = "std")]
+use core::fmt;
+use core::ops::{Deref, DerefMut};
+#[cfg(feature = "std")]
+use std::io::{IoSlice, Result, Write};
+
+use super::Utf8String;
+
+#[cfg(feature = "std")]
+impl Write for Utf8String {
```

Comment:
Split `io::Write` implementations for all encoded string types into an `io` module

This will make the feature flagging a bit clearer.

Reply:
Filed:

- https://github.com/artichoke/artichoke/issues/1724